pkgs/by-name/ya/yazi-unwrapped/package.nix

```
@@ -26,14 +26,15 @@ let
     hash = "sha256-kEVXejDg4ChFoMNBvKlwdFEyUuTcY2VuK9j0PdafKus=";
   };
 in
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "yazi";
   inherit version;
 
   srcs = [
     code_src
     man_src
   ];
```

Non-blocking: Maybe it should be `rustPlatform.buildRustPackage (finalAttrs: { ... })` and then this can be

```suggestion
  srcs = builtins.attrValues finalAttrs.passthru.srcs;
```

?

---

Didn't know `buildRustPackage` supported `finalAttrs`, nice! That's definitely better.

---

Yeah, it's super recent: #382550